mailboxes/lang/de

```diff
@@ -203,12 +203,12 @@ reply_mailforward=Weitergeleitete Mails
 reply_attach2=Client- und serverseitige Anhänge
 reply_attach3=Hochgeladene Anhänge
 reply_send=Mail versenden
-reply_ecannot=Sie dürfen Mails im Namen dieses Benutzers nicht versenden
+reply_ecannot=Sie dürfen Mails im Namen dieses Benutzers nicht versenden.
```

Comment:
Why adding this `dot` and many other alike?

Reply:
> you really need to come up with a guideline on how you expect translations to happen. Otherwise, you put off people coming up with the rules after PR got submitted.

Different phrases are used in different contexts, and context defines everything. For example, you don't use a full stop (dot) in buttons or similar scenarios. It won't be easy, or even possible, to create a clear guideline because there are too many variables that can't be understood just by looking at the text file alone.

However, as [mentioned](https://forum.virtualmin.com/t/translation-rules-policies-guidelines/128961?u=ilia) on the forum, we've made everything ready and simple for those who want to contribute. Every `.auto` file already contains everything formatted correctly. The only fix needed is for the language itself, not the formatting. The `.auto` files are generated automatically using the `language-manager` script, which inherits everything from the source English language file.

Let's please keep it simple and not overthink the original problem.

Wer Wind sät, wird Sturm ernten. 😊